bootstrap/src/main/scala/org/scalafmt/bootstrap/Bootstrap.scala

```diff
@@ -77,7 +77,7 @@ object ScalafmtBootstrap {
         logger = Some(logger)
       )
     )
-    val resolution = start.process.run(fetch).unsafePerformSync
```

Comment:
Actually, it's probably enough to put a `synchronized` on the fromVersionUncached` method.